clients/admin-ui/src/features/dataset/DatabaseConnectForm.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Stack, Text } from "@fidesui/react";
+import { Form, Formik } from "formik";
+import * as Yup from "yup";
+
+import { CustomSelect, CustomTextInput } from "../common/form/inputs";
+import { DATABASE_OPTIONS } from "./constants";
+
+const initialValues = { type: "", url: "" };
+
+const ValidationSchema = Yup.object().shape({
+  type: Yup.string().required().label("Database type"),
+  url: Yup.string().required().label("Database URL"),
+});
+
+const DatabaseConnectForm = () => {
+  const handleCreate = () => {
+    //   TODO when backend supports this (829)
+  };
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      validationSchema={ValidationSchema}
+      onSubmit={handleCreate}
+      validateOnChange={false}
+      validateOnBlur={false}
+    >
+      {({ isSubmitting }) => (
+        <Form>
+          <Text size="sm" color="gray.700" mb={8}>
+            Connect to one of your databases using a connection URL. You may
+            have received this URL from a colleague or your Ethyca developer
+            support engineer.
+          </Text>
+          <Stack mb={8} spacing={4}>
+            <CustomSelect
+              name="type"
+              label="Database type"
+              options={DATABASE_OPTIONS}
+            />
+            <CustomTextInput name="url" label="Database URL" />
```

Comment:
How come we take in the database type when that is included in the connection url? I think it would be somewhat redundant and not actually used. Here is what a connection string looks like right now:
`postgresql+psycopg2://postgres:postgres@postgres-test:5432/postgres_example`

I wonder if this UI should have an option for building the url or submitting a custom url.

`postgresql+psycopg2://postgres:postgres@postgres-test:5432/postgres_example`

or

```
db type = postgresql
host = postgres-test
port = 5432
username = postgres
password = postgres
db name = postgres_example
```

Reply:
UNLESS, we wanted connection url to be everything but the driver and we expected this input:
`postgres:postgres@postgres-test:5432/postgres_example`

Then the frontend or backend picks the driver based on database type?

Reply:
oh yeah, that's a great point. I thought perhaps the backend needed it, but just having a connection string seems like it'd work. The most user friendly thing might be what you suggested with db type, host, port, etc. Might need to go back to the designs on this... what do you think @mfbrown ?

Reply:
having them provide the connection string requires the user to be aware of unnecessary implementation details, it's much more elegant to me to have them provide the separate components that we can then use to build the connection string.

Doing this also gives us the flexibility to choose if we want to build a synchronous or asynchronous connection string

Reply:
perhaps for now we ask for the connection string, and then have a separate line of work to separate out the components to build the string? I imagine there'd be extra stuff we'd have to build in order to compose the string properly for different database types? 

so what I would propose for this PR is to remove the database type field, and just leave a field for connection string, then open up a ticket to investigate if we can make this UI better by building the connection string for the user

Reply:
Sounds good to me!